CONTRIBUTING_JS.md

```diff
@@ -132,9 +132,14 @@ Always run tests before pushing and PR'ing your code.
 
 #### Supported Versions
 
-IPFS projects support both the Current and Active LTS versions of Node.js.  Please see [nodejs.org](https://nodejs.org/) for what these currently are.
+The IPFS JavaScript projects follow the LTS of Node.js and respective npm version that gets installed with Node.js. Please consult [nodejs.org](https://nodejs.org/) for LTS timeline. Today these are:
```

Comment:
I don't find this clear enough. What is "LTS of Node.js"? Reading the linked issue I think you mean "only the most recent active LTS of Node.js"?

Reply:
Good point, I've updated it now :)